src/CentrifugeRouter.sol

```diff
@@ -71,6 +74,12 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         emit LockDepositRequest(vault, controller, owner, _initiator, amount);
     }
 
+    /// @inheritdoc ICentrifugeRouter
+    function openLockDepositRequest(address vault, uint256 amount) external protected {
```

Comment:
Just a helper to allow Prime users that interact with Gnosis Safe directly, to easily construct this transaction.